packages/core/core/src/worker.js

```diff
@@ -50,6 +52,8 @@ function loadOptions(ref, workerApi) {
 }
 
 async function loadConfig(cachePath, options) {
+  await initSourcemaps;
+  await initHash;
```

Comment:
Should we enable workers to export an async init function that is called here:
https://github.com/parcel-bundler/parcel/blob/efc6cdb72af26d72bb276cb081c8f87db778c23e/packages/core/workers/src/child.js#L130-L142
?
That is both cleaner and might also be faster (currently these two awaits of resolved Promises would be happen on almost every request that is sent to workers)